src/test/integration/things-test.js

```diff
@@ -396,20 +396,6 @@ describe('things/', function() {
       }
     }
     expect(!found).assert('should not find thing in /things output');
-
-    res = await chai.request(server)
```

Comment:
Why isn't the thing expected here? It looks like it should still be there but I might be wrong about how mock-adapter works

Reply:
There was previously some weirdness in that things could be present in the adapters but not in the gateway. Due to the changes in this patch set, that is no longer the case.